src/commands/live.rs

```diff
@@ -2,15 +2,47 @@ use super::{format_hours, local_time_format};
 use super::{Error, Result};
 use crate::client::{break_policy, time_entries};
 use spinner_macro::spinner_wrap;
+use std::time::Duration;
 
 #[spinner_wrap]
 pub fn status() -> Result<String> {
     Ok(match time_entries::current_entry(&super::get_session())? {
-        Some(entry) => format!("Clocked in since {}!", local_time_format(entry.start_time)),
+        Some(entry) => {
+            let mut msg = format!("Clocked in since {}", local_time_format(entry.start_time));
+
+            let regular_hours_in_seconds = (entry.regular_hours * 3600.0) as u64;
+            let regular_hours_formatted = format_seconds_as_human_readable(regular_hours_in_seconds);
+
+            let total_duration_in_seconds = (entry.duration * 3600.0) as u64;
+            let breaks_in_seconds = total_duration_in_seconds - regular_hours_in_seconds;
```

Comment:
Instead of calculating this you should use `unpaidBreakHours`, https://github.com/mkon/rippling-cli/blob/main/tests/fixtures/time_entry.json#L54